packages/xdl/src/Utils.ts

```diff
@@ -44,3 +49,22 @@ export class Semaphore {
     }
   }
 }
+
+export class XdlSchemaClient {
```

Comment:
A better name for this would be `ConfigurationSchemaClient`. (The old method name `xdlSchemaAsync`  was incorrect, or at least outdated by now.)

But since this "singleton" only has static properties and this is not Java, you might as well remove the class and have a function like `getConfigurationSchemaAsync`. (`_schemaCaches` can be a top-level variable in the module.) Please also move it to `ExpSchema.ts`, `Utils` only has some helper functions that are not really bound to our APIs or anything like that.